requirements.txt

```diff
@@ -1,15 +1,12 @@
 numpy
 accelerate>=0.20.3
-packaging
-transformers>=4.34.1
+transformers==4.37.2
```

Comment:
Seems @fabianlim has resolved the xla_fsdp_v2 issue [PR](https://github.com/foundation-model-stack/fms-hf-tuning/pull/53).

Reply:
I can try to rebase, make the change and see if I can get it to work this afternoon

Reply:
Yes, I see that @fabianlim pr has been merged into this, and now the transfomers dependency can be open... it's working with the latest version.  Thank you @tedhtchang 
```
pip list |grep transformers
transformers             4.38.2
```
and it allows the fms-hf-tuning to install and run:
```
pip list |grep fms
fms-hf-tuning            0.0.1       /root/JIM/fms-hf-tuning
```